--- docker-compose.yml	2025-02-08 21:41:16.170778265 +0100
+++ docker-compose.yml.www-with-app	2025-02-08 21:41:37.944903088 +0100
@@ -32,20 +32,10 @@
         cap_add:
             - DAC_READ_SEARCH
         # tty: true
-    www:
+    wwwapp:
         build:
             context: ./src
-            dockerfile: Dockerfile.www
-            # By default, Dockerfile.www produces application backend in
-            # separate container. To run the example application under
-            # mod_wsgi directly in the www container, comment out the
-            # above dockerfile line and uncomment the following one.
-            #
-            # dockerfile: Dockerfile.www-with-app
-            #
-            # If you use Dockerfile.www-with-app, the link to app
-            # service and the whole app service definition below can
-            # be commented out as well.
+            dockerfile: Dockerfile.www-with-app
         volumes:
             - ./www-data:/data:Z
             # - /sys/fs/cgroup:/sys/fs/cgroup:ro
@@ -71,12 +61,3 @@
             - "55022:22"
         cap_add:
             - DAC_READ_SEARCH
-    app:
-        build:
-            context: ./src
-            # dockerfile: Dockerfile.django-dist
-            dockerfile: Dockerfile.django-pip
-        volumes:
-            - ./app-data:/data:Z
-        hostname: app.example.test
-        stop_signal: KILL
