--- docker-compose.yml	2025-09-17 08:56:14.872011998 +0200
+++ docker-compose.yml.www-with-app	2025-09-17 08:57:07.134876333 +0200
@@ -37,17 +37,10 @@
         cap_add:
             - DAC_READ_SEARCH
         # tty: true
-    www:
+    wwwapp:
         build:
             context: ./src
-            dockerfile: Dockerfile.www
-            # By default, Dockerfile.www produces application backend in
-            # separate container. To run the example application under
-            # mod_wsgi directly in the www container, comment out the
-            # above dockerfile line and uncomment the following one.
-            #
-            # dockerfile: Dockerfile.www-with-app
-            #
+            dockerfile: Dockerfile.www-with-app
         volumes:
             - ./www-data:/data:Z
         tmpfs:
@@ -77,12 +70,3 @@
             - "55022:22"
         cap_add:
             - DAC_READ_SEARCH
-    app:
-        build:
-            context: ./src
-            # dockerfile: Dockerfile.django-dist
-            dockerfile: Dockerfile.django-pip
-        volumes:
-            - ./app-data:/data:Z
-        hostname: app.example.test
-        stop_signal: KILL
