--- docker-compose.yml	2025-01-07 13:24:30.383332565 +0100
+++ docker-compose.yml.www-with-app	2025-01-07 13:45:34.152136298 +0100
@@ -34,20 +34,10 @@
         cap_add:
             - DAC_READ_SEARCH
         # tty: true
-    www:
+    wwwapp:
         build:
             context: ./src
-            dockerfile: Dockerfile.www
-            # By default, Dockerfile.www produces application backend in
-            # separate container. To run the example application under
-            # mod_wsgi directly in the www container, comment out the
-            # above dockerfile line and uncomment the following one.
-            #
-            # dockerfile: Dockerfile.www-with-app
-            #
-            # If you use Dockerfile.www-with-app, the link to app
-            # service and the whole app service definition below can
-            # be commented out as well.
+            dockerfile: Dockerfile.www-with-app
         volumes:
             - ./www-data:/data:Z
             # - /sys/fs/cgroup:/sys/fs/cgroup:ro
@@ -56,7 +46,6 @@
         links:
             - ipa:ipa.example.test
             - idp:idp.example.test
-            - app:app.example.test
         cap_add:
             - DAC_READ_SEARCH
         # Uncomment the following if you want to be able to access the
@@ -76,17 +65,8 @@
         links:
             - ipa:ipa.example.test
             - idp:idp.example.test
-            - www:www.example.test
+            - wwwapp:www.example.test
         ports:
             - "55022:22"
         cap_add:
             - DAC_READ_SEARCH
-    app:
-        build:
-            context: ./src
-            # dockerfile: Dockerfile.django-dist
-            dockerfile: Dockerfile.django-pip
-        volumes:
-            - ./app-data:/data:Z
-        hostname: app.example.test
-        stop_signal: KILL
