--- /usr/lib/python3.6/site-packages/ipaclient/install/client.py	2018-07-19 16:58:10.000000000 +0000
+++ /usr/lib/python3.6/site-packages/ipaclient/install/client.py	2018-12-08 06:29:30.115487176 +0000
@@ -673,7 +673,7 @@
         os.path.basename(paths.KRB5_FREEIPA) + ".template"
     )
     shutil.copy(template, paths.KRB5_FREEIPA)
-    os.chmod(paths.KRB5_FREEIPA, 0x644)
+    os.chmod(paths.KRB5_FREEIPA, 0o644)
 
     # Then, perform the rest of our configuration into krb5.conf itself.
     krbconf = IPAChangeConf("IPA Installer")
