--- /usr/lib/python3.12/site-packages/ipsilon/tools/dbupgrade.py	2022-07-06 23:12:19.000000000 +0200
+++ /usr/lib/python3.12/site-packages/ipsilon/tools/dbupgrade.py	2023-12-09 11:28:48.909577153 +0100
@@ -82,8 +82,7 @@
         loader=FileSystemLoader(
             os.path.join(cherrypy.config['base.dir'],
                          'templates')),
-        autoescape=True,
-        extensions=['jinja2.ext.autoescape'])
+        autoescape=True)
     root = Root('default', template_env)
 
     # Handle the session store if that is Sql
--- /usr/libexec/ipsilon/ipsilon	2023-08-12 02:00:00.000000000 +0200
+++ /usr/libexec/ipsilon/ipsilon	2023-12-09 11:28:52.697502930 +0100
@@ -73,8 +73,7 @@
     os.path.join(cherrypy.config['base.dir'],
                  default_template_dir)))
 template_env = Environment(loader=ChoiceLoader(template_loaders),
-                           autoescape=True,
-                           extensions=['jinja2.ext.autoescape'])
+                           autoescape=True)
 
 transchars = string.punctuation.replace('-', '').replace('.', '')
 if six.PY2:
